src/manual/peer2peer/js/main.js

```diff
@@ -378,24 +379,45 @@ function getDevices() {
   checkIfDeviceDropdownsArePopulated_();
 }
 
+function displayScreenCaptureInfo() {
+  if ($('screencapture-info')) {
+    $('screencapture-info').style.display = 'block';
+    return;
+  }
+  var message = 'Please install the screen capture extension:<br>' +
+                '1. Go to chrome://extensions<br>' +
+                '2. Check: "Enable Developer mode"<br>' +
+                '3. Click: "Load the unpacked extension..."<br>' +
+                '4. Choose "extension" folder from the ' +
+                '<a href="http://goo.gl/kLEycY">repository</a><br>' +
+                '5. Reload this page over https<br>' +
+                'Note: Make sure the URL permission in manifest.json matches ' +
+                'the URL for this page.';
+  var startScreenCaptureButton = document.getElementById('start-screencapture');
+  var messageDiv = document.createElement('div');
+  messageDiv.innerHTML = message;
+  messageDiv.id = 'screencapture-info';
+
+  window.onclick = function(event) {
+    if (event.target === messageDiv) {
+      messageDiv.style.display = 'none';
+    }
+  };
+
+  document.getElementById('general-gum').insertBefore(messageDiv,
+    startScreenCaptureButton);
+}
+
 function screenCaptureExtensionHandler_() {
-  // Copied and modifed from desktopcapture example.
+  // Copied and modified from desktop capture example.
   var extensionInstalled = false;
-  document.getElementById('start-screencapture').addEventListener('click',
-      function() {
-        // send screen-sharer request to content-script
-        if (!extensionInstalled) {
-          var message = 'Please install the extension:\n' +
-                        '1. Go to chrome://extensions\n' +
-                        '2. Check: "Enable Developer mode"\n' +
-                        '3. Click: "Load the unpacked extension..."\n' +
-                        '4. Choose "extension" folder from the repository:\n' +
-                        '(Download from here: http://goo.gl/kLEycY)\n' +
-                        '5. Reload this page over https';
-          alert(message);
-        }
-        window.postMessage({type: 'SS_UI_REQUEST', text: 'start'}, '*');
-      });
+  $('start-screencapture').addEventListener('click', function() {
+    // send screen-sharer request to content-script
+    window.postMessage({type: 'SS_UI_REQUEST', text: 'start'}, '*');
+    if (!extensionInstalled) {
```

Comment:
It's part of an event listener, it's set to true when a ping response from the extension is received https://github.com/webrtc/testrtc/pull/222/files/70b11656f4a3fc14dc977a69193bd86d11f4c06c#diff-39bad8872c33a8321fc56bcb9eee5587R430